assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -5,9 +5,12 @@
 	}
 }
 
+.e-container {
+	--container-widget-width: 100%;
+}
```

Comment:
I noticed an error in the screenshots. I realised we should apply the css variable to the `.e-container__inner`, because that overwrites the css variable generated by the Container direction control.